README.md

```diff
@@ -91,3 +91,35 @@ generation. In that case we excluded the file from being generated and manually
 `crates/bitwarden-api-api/src/models/organization_user_status_type.rs`
 
 The hope going forward is that we can continue to use the generator with minimal manual intervention.
+
+### WASM - WebAssembly
+
+Install `wasm32-unknown-unknown` rust target.
+
+```bash
+rustup target add wasm32-unknown-unknown
+```
+
+Install wasm-bindgen-cli
+
+```bash
+cargo install -f wasm-bindgen-cli
```

Comment:
I had to separately install `wasm-opt`, used `brew install binaryen` on Mac, or from the releases page: https://github.com/WebAssembly/binaryen/releases. Probably a good idea to mention it here as well